packages/istanbul-lib-instrument/test/varia.test.js

```diff
@@ -47,6 +47,15 @@ describe('varia', function () {
         assert.ok(code.match(/\/* hello */));
     });
 
+    it('preserves function names for named export arrow functions', function () {
+        /* https://github.com/istanbuljs/babel-plugin-istanbul/issues/125 */
+        var v = verifier.create('export const func = () => true;', { generateOnly: true }, { esModules: true }),
```

Comment:
I'm glad you dug up this pattern of testing features not supported by v8 inside a miscellaneous testing file, let's stick with it.